### PR TITLE
update website 22318

### PR DESCRIPTION
**Pull request by jburgler**

### CODE REVIEW

**Comment by jburgler** [comment]
@ahoefling

File: _data/index/schedule.yml [hunk 1]
```diff
@@ -1,59 +1,69 @@
...
     detail: 'Philadelphia, PA'
   icon: fa-plus-square
 
 - name:
-    detail: 'Flour City Fear vs Shenanigans'
+    detail: 'Flour City Fear vs Blue Streaks'
     i18n: company_a
   desc:
-    detail: 'Regulation Game against PennJersey Hooligans B-Team'
+    detail: 'Regulation Game against Pittsburgh Blue Streaks'
```
Comment (SkyeHoefling) [inline _data/index/schedule.yml:17]: Is this a regulation game or is that a typo from the old result?

**Comment by SkyeHoefling** [comment]
Everything looks good here, I had one comment about something that may be a typo. Good job!!

Once you verify that change we can merge this in and it will update the website

**Comment by jburgler** [comment]
@ahoefling im not seeing any comments? where is the issue

**Comment by SkyeHoefling** [comment]
check now

File: _data/index/schedule.yml (hunk 1 above)
Comment (jburgler) [reply _data/index/schedule.yml:17]: I believe it is regulation, I havent heard otherwise

Comment (jburgler) [reply _data/index/schedule.yml:17]: @ahoefling I would say merge for now and if i have to change it I can. I need to update the website again once voting ends for new board members anyway

**Comment by SkyeHoefling** [comment]
Looks there there was a build error, see the latest commit it is talking about the schedule.yml file on line 44. Can you take a look at this and see what's up

**Comment by jburgler** [comment]
Should be fixed, new pr is up

Thank you,
-Justin J Burgler

On Feb 26, 2018, at 10:47 AM, Andrew Hoefling <notifications@github.com<mailto:notifications@github.com>> wrote:


Looks there there was a build error, see the latest commit it is talking about the schedule.yml file on line 44. Can you take a look at this and see what's up

**Pull request merged**